model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -31,20 +31,44 @@ data class SpdxCompoundExpression(
         val left: SpdxExpression,
         val operator: SpdxOperator,
         val right: SpdxExpression
-) : SpdxExpression()
+) : SpdxExpression() {
+    override fun toString(): String {
```

Comment:
BTW, @fviernau and I were discussing whether we should really override `toString()` in this case, as we're trying to not do this anymore in other code areas to get rid of hard-to-discover implicit calls in strings. So far we're a bit undecided, this seems to be an edge-use-case.

Reply:
Can we discuss this in person next week? We would have to implement a replacement for the `ToStringDeserializer` at least and do the same for `Identifier`.

Reply:
Yes, I have some other "cleanup"-related topics to discuss as well. The question is, should we wait with the merge until next week, or merge now and eventually fix it later?

Reply:
Please let us merge today as I want to build more on top of this.